lib/profile.dart

```diff
@@ -0,0 +1,150 @@
+import 'package:flutter/material.dart';
+import 'pages/data.dart';
+
+class ProfilePage extends StatefulWidget {
+  const ProfilePage({super.key, required this.title});
+
+  final String title;
+
+  @override
+  State<ProfilePage> createState() => _ProfilePageState();
+}
+
+
+class _ProfilePageState extends State<ProfilePage> {
+
+  String User_name_data = '読み込み中...';
+  int grade = 0;
+  String User_info = '読み込み中...';
+  String User_link = '読み込み中...';
```

Comment:
- 変数名のつけ方が一般的な感じじゃないので、一般的なやり方に合わせてほしい
- `User_name_data`は`User_name`ではだめだったの...？
- `grade`は他に合わせて`userGrade`の方がいいんじゃないかな
- これらは同じ目的のデータだと思うから、クラスにまとめた方がいいかも

```suggestion
  String userName = '読み込み中...';
  int grade = 0;
  String userInfo = '読み込み中...';
  String userLink = '読み込み中...';
```

とりあえず1つ目だけ修正したのをsuggestする

Reply:
全部変更しました！